docs/madr/decisions/070-resource-identifier.md

```diff
@@ -0,0 +1,380 @@
+|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
+| Inbound Listener  | `inbound:10.43.205.116:8080`<br>`inbound:[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:8080`                                                                                                                | Dataplane (with sectionName to select port)                                                                                              | kri_dp_mesh-1_us-east-2_kuma-demo_backend-app_8080   |
+| Outbound Listener | `outbound:10.43.205.116:8080`<br>`outbound:[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:8080`                                                                                                              | Mesh*Service (with sectionName to select port)                                                                                           | kri_msvc_mesh-1_us-east-2_kuma-demo_backend_httpport |
+| VirtualHost       | legacy listeners - `<kuma.io/service>`<br>new outbounds - `<mesh>_<name>_<namespace>_<zone>_<short-name>_<port>`                                                                                        | Mesh*Service (with sectionName to select port)                                                                                           | kri_msvc_mesh-1_us-east-2_kuma-demo_backend_httpport |
+| Inbound Cluster   | `localhost:<port>`                                                                                                                                                                                      | Dataplane (with sectionName to select port)                                                                                              | kri_dp_mesh-1_us-east-2_kuma-demo_backend-app_8080   |
+| Outbound Cluster  | legacy clusters - `<kuma.io/service>-hash(dst.tags)`<br>legacy clusters cross-mesh - `<kuma.io/service>-hash(dst.tags)_<mesh>`<br>new clusters - `<mesh>_<name>_<namespace>_<zone>_<short-name>_<port>` | Mesh*Service (with sectionName to select port)                                                                                           | kri_msvc_mesh-1_us-east-2_kuma-demo_backend_httpport |
+| Route             | Routes are set on Listener on VirtualHost.<br>On inbound - `inbound:<kuma.io/service>`<br>On outbound - `<hash_sha256([]Match{...})>`                                                                   | Envoy Route is a merge product of multiple MeshHTTPRoutes.<br/> We can use only the most specific MeshHTTPRoute policy to name the route | kri_mhttpr_mesh-1_us-east-2_kuma-demo_route-1_       |
```

Comment:
What will be the name for default routes, since we don't need a policy to get ones?

Reply:
We probably should treat it as an internal entity, so it has to start with `_kuma`, I'll update the doc

Reply:
How would it look like? Since, we might have a lot of default routes it's hard to say

Reply:
What do you mean by "a lot of default routes"? Don't we have just one [catchAll](https://github.com/kumahq/kuma/blob/820494cf59e22d5a4870ca6d7a01f1f77e38ad46/pkg/plugins/policies/meshhttproute/plugin/v1alpha1/listeners.go#L196) route? It probably can be named something like `_kuma_catchAll` but I don't want to be too specific in this document about the way we name internal entities, it should be out of the scope, I left a note:
> Following the  [Internal listeners naming convention](036-internal-listeners.md) we need to rename internal listeners and clusters to start with `_`. Additionally, we need to get rid of `:` in internal entities names and use `_` instead.